src/components/Content.jsx

```diff
@@ -1,20 +1,26 @@
 import { useState, useEffect } from "react";
-import { loadData, loadComments } from "./loadData";
+import { loadData, loadComments, loadUsers } from "./loadData";
 
 const Content = () => {
   const [posts, setPosts] = useState([]);
   const [error, setError] = useState(null);
+  const [users, setUsers] = useState([]);
 
   const addComments = (loadedPosts) => {
     loadComments(loadedPosts).then((postsWithComments) => {
       setPosts(postsWithComments)
     }).catch(setError);
   };
 
+  const addUsers = (loadedPosts) => {
+    loadUsers(loadedPosts).then(setUsers).catch(setError);
+  }
+
   useEffect(() => {
     loadData().then((loadedPosts) => {
       setPosts(loadedPosts);
       addComments(loadedPosts);
+      addUsers(loadedPosts);
```

Comment:
The suggestion here is the same from task 2.1: we could avoid setting the state before we have all the data and move the call for `addUsers` to the file `loadData.js`.